src/analyze.ts

```diff
@@ -1196,6 +1196,14 @@ export default async function analyze(
       return;
     }
 
+    if (
+      'value' in staticChildValue &&
+      typeof staticChildValue.value === 'string' &&
+      staticChildValue.value.startsWith('.')
```

Comment:
Since this code is checking for `.`, I think we need a few more tests.

For example:

```js
readFile('../foo.txt')
readFile('../../bar.txt')
readFile('../bar/../baz.txt')
```

Reply:
Could we flip this though? The branch under checks `isAbsolutePathOrUrl`, if we fail that check, we could enter an `else if('value' in staticChildValue &&
      typeof staticChildValue.value === 'string)` branch where we try to resolve if `cwd` + the static value exists, and if so trace it.